src/faq.njk

```diff
@@ -165,12 +166,31 @@ eleventyNavigation:
 
   <main>
     <dl class="faq__faq">
-      <dt>Why won't it start in VirtualBox?</dt>
+      
+      <dt>I downloaded the developer preview liveCD. What is the username and password?</dt>
+      <dd class="faq__markdown">
+        <p>The login credentials for the liveboot currently is <em>liveuser</em>, all in small letters, and <b>no password</b>. Just type <em>liveuser</em> in the username field and press enter/click on the login button.</p>
+      </dd>
+
+      <dt>Why won't it display GUI in VirtualBox/Vmware/QEMU?</dt>
       <dd class="faq__markdown">
         <p>
-              Release 0.2.2 may hang after booting the LiveCD with a black screen if you run in VirtualBox with the default VMSVGA driver. Changing to the VBoxSVGA driver for display may fix the issue. See <a href=https://github.com/ravynsoft/ravynos/issues/74>issue #74</a> for details.
-            </p>
+              Since ravynOS 0.4.0 is having its GUI written from scratch, there is no driver for any virtual machine GPU yet, including VMware VGA, Virtualbox SVGA, virtio-vga or qxl and thus, no form of graphics acceleration is currently available. The OS will display a shell-like/ tty interface on virtual machines. 
+        </p>      
+        <p>
+            We are working on this, and help is welcome. There is work being done on vmwgfx for ravynOS and this might be released soon, making it possible to test LiveCD on virtual machines 🙂. Currently, VMs are usable for non-UI development.
+        </p>
       </dd>
+
+
+      <dt>When I try booting the LiveCD for ravynOS, it shows a blank screen on my PC. Why is that so?</dt>
+      <dd class="faq__markdown">
+        <p>
+              Video drivers, for the time being, exist for only Intel HD series, Intel Iris series (Xe graphics), most AMD integrated and discrete GPUs. There is no driver yet for NVIDIA. Help fixing this is most welcome.  If your GPU is supported, the GUI starts automatically on boot. If not, you will receive get boot messages on screen followed by a shell-like interface (<b>tty1</b>).
```

Comment:
typo here: "receive get" ?

Reply:
Didn't see that one. Apologies :sweat_smile:  !!